### PR TITLE
mod mindpx v2 prototype and defconfig

### PR DESCRIPTION
**Pull request by eflyinghi**
According to the change of Bootloader, mod mindpx v2 prototype and defconfig.

Please refer to https://github.com/PX4/Bootloader/pull/54

@LorenzMeier please help re-assign the board_id for both this PR and Bootloader.  Thanks.

### CODE REVIEW

**Comment by iZhangHui** [comment]
+CONFIG_CDCACM_PRODUCTSTR="MindPX BL v2.x" 这个地方改的不对


MindPX FMU v2.x



2016-05-04 15:39 GMT+08:00 eFlyingHi <notifications@github.com>:

File: nuttx-configs/mindpx-v2/nsh/defconfig [hunk 1]
```diff
@@ -667,9 +667,9 @@ CONFIG_CDCACM_BULKIN_REQLEN=96
 CONFIG_CDCACM_RXBUFSIZE=1000
 CONFIG_CDCACM_TXBUFSIZE=8000
 CONFIG_CDCACM_VENDORID=0x26ac
-CONFIG_CDCACM_PRODUCTID=0x0011
+CONFIG_CDCACM_PRODUCTID=0x0030
 CONFIG_CDCACM_VENDORSTR="3D Robotics"
-CONFIG_CDCACM_PRODUCTSTR="PX4 FMU v2.x"
+CONFIG_CDCACM_PRODUCTSTR="MindPX BL v2.x"
```
Comment (iZhangHui) [inline nuttx-configs/mindpx-v2/nsh/defconfig]: Should be changed to CONFIG_CDCACM_PRODUCTSTR="MindPX FMU v2.x"

Comment (eflyinghi) [inline nuttx-configs/mindpx-v2/nsh/defconfig]: Changed, thanks.

**Comment by LorenzMeier** [comment]
Rebased and applied, BL change is in as well.

**Pull request closed without merging**